### ISSUE TITLE
`fontist status arial` shows "Arial" not found on macOS even when shown in `fontist install arial` command

### ISSUE THREAD

**Issue by ronaldtse** [fontist/fontist#209]
From https://github.com/metanorma/metanorma-cli/issues/217 by @gbuhler:

> I was getting an error on my Mac running the Metanorma scripts.
> 
> ```
> /usr/local/bundle/gems/fontist-1.8.4/lib/fontist/index.rb:7:in `from_yaml': Please fetch index with `fontist update`.
> (Fontist::Errors::FormulaIndexNotFoundError)
> ```
> 
> I found where it seems you were having the same type of issue:
> 
> https://github.com/fontist/fontist/issues/177
> 
> I have installed and updated the fontist gem, but it is missing Arial.

Output:
```sh
Greg: greg$ fontist install 'arial'
Fonts found at:
- /Library/Fonts/Arial Bold.ttf
- /Library/Fonts/Arial.ttf
- /Library/Fonts/Arial Bold Italic.ttf
- /Library/Fonts/Arial Italic.ttf
- /Applications/Microsoft Word.app/Contents/Resources/DFonts/arialbd.ttf
- /Applications/Microsoft Word.app/Contents/Resources/DFonts/arial.ttf
- /Applications/Microsoft Word.app/Contents/Resources/DFonts/ariblk.ttf
- /Applications/Microsoft Word.app/Contents/Resources/DFonts/ariali.ttf
- /Applications/Microsoft Word.app/Contents/Resources/DFonts/arialbi.ttf
- /Applications/Microsoft Excel.app/Contents/Resources/DFonts/arialbd.ttf
- /Applications/Microsoft Excel.app/Contents/Resources/DFonts/arial.ttf
- /Applications/Microsoft Excel.app/Contents/Resources/DFonts/ariblk.ttf
- /Applications/Microsoft Excel.app/Contents/Resources/DFonts/ariali.ttf
- /Applications/Microsoft Excel.app/Contents/Resources/DFonts/arialbi.ttf
- /Applications/Microsoft Outlook.app/Contents/Resources/DFonts/arialbd.ttf
- /Applications/Microsoft Outlook.app/Contents/Resources/DFonts/arial.ttf
- /Applications/Microsoft Outlook.app/Contents/Resources/DFonts/ariblk.ttf
- /Applications/Microsoft Outlook.app/Contents/Resources/DFonts/ariali.ttf
- /Applications/Microsoft Outlook.app/Contents/Resources/DFonts/arialbi.ttf
- /Applications/Microsoft OneNote.app/Contents/Resources/DFonts/arialbd.ttf
- /Applications/Microsoft OneNote.app/Contents/Resources/DFonts/arial.ttf
- /Applications/Microsoft OneNote.app/Contents/Resources/DFonts/ariblk.ttf
- /Applications/Microsoft OneNote.app/Contents/Resources/DFonts/ariali.ttf
- /Applications/Microsoft OneNote.app/Contents/Resources/DFonts/arialbi.ttf
- /Applications/Microsoft PowerPoint.app/Contents/Resources/DFonts/arialbd.ttf
- /Applications/Microsoft PowerPoint.app/Contents/Resources/DFonts/arial.ttf
- /Applications/Microsoft PowerPoint.app/Contents/Resources/DFonts/ariblk.ttf
- /Applications/Microsoft PowerPoint.app/Contents/Resources/DFonts/ariali.ttf
- /Applications/Microsoft PowerPoint.app/Contents/Resources/DFonts/arialbi.ttf
Greg: greg$ fontist status arial
'arial' font is missing, please run `fontist install 'arial'` to download the font.
```

Obviously, that did not work.

```
================
FONT LICENSE ACCEPTANCE REQUIRED FOR "Arial":

Fontist can install this font if you accept its licensing conditions.

FONT LICENSE BEGIN ("Arial")
-----------------------------------------------------------------------
(from http://web.archive.org/web/20011222020409/http://www.microsoft.com/typography/fontpack/eula.htm)
TrueType core fonts for the Web EULA...
```

Here is the output from running command:
```
docker run -v "$(pwd)":/metanorma metanorma/mn  metanorma compile -t ogc -x xml,html,doc,pdf document.adoc
-----------------------------------------------------------------------

FONT LICENSE END ("Arial")
/usr/local/bundle/gems/fontist-1.8.4/lib/fontist/font.rb:120:in `check_and_confirm_required_license': Fontist will not download these fonts unless you accept the terms. (Fontist::Errors::LicensingError)
from /usr/local/bundle/gems/fontist-1.8.4/lib/fontist/font.rb:105:in `download_font'
from /usr/local/bundle/gems/fontist-1.8.4/lib/fontist/font.rb:42:in `install'
from /usr/local/bundle/gems/fontist-1.8.4/lib/fontist/font.rb:22:in `install'
from /usr/local/bundle/gems/fontist-1.8.4/lib/fontist/manifest/install.rb:23:in `install_font'
from /usr/local/bundle/gems/fontist-1.8.4/lib/fontist/manifest/install.rb:17:in `file_paths'
from /usr/local/bundle/gems/fontist-1.8.4/lib/fontist/manifest/locations.rb:51:in `block in style_paths'
from /usr/local/bundle/gems/fontist-1.8.4/lib/fontist/manifest/locations.rb:50:in `map'
from /usr/local/bundle/gems/fontist-1.8.4/lib/fontist/manifest/locations.rb:50:in `style_paths'
from /usr/local/bundle/gems/fontist-1.8.4/lib/fontist/manifest/locations.rb:45:in `style_paths_map'
from /usr/local/bundle/gems/fontist-1.8.4/lib/fontist/manifest/locations.rb:40:in `block in font_paths'
from /usr/local/bundle/gems/fontist-1.8.4/lib/fontist/manifest/locations.rb:38:in `each'
from /usr/local/bundle/gems/fontist-1.8.4/lib/fontist/manifest/locations.rb:38:in `map'
from /usr/local/bundle/gems/fontist-1.8.4/lib/fontist/manifest/locations.rb:38:in `font_paths'
from /usr/local/bundle/gems/fontist-1.8.4/lib/fontist/manifest/locations.rb:26:in `call'
from /usr/local/bundle/gems/fontist-1.8.4/lib/fontist/manifest/locations.rb:21:in `from_hash'
from /usr/local/bundle/gems/metanorma-1.2.4/lib/metanorma/compile.rb:312:in `fontist_install'
from /usr/local/bundle/gems/metanorma-1.2.4/lib/metanorma/compile.rb:308:in `rescue in install_fonts_safe'
from /usr/local/bundle/gems/metanorma-1.2.4/lib/metanorma/compile.rb:292:in `install_fonts_safe'
from /usr/local/bundle/gems/metanorma-1.2.4/lib/metanorma/compile.rb:287:in `install_fonts'
from /usr/local/bundle/gems/metanorma-1.2.4/lib/metanorma/compile.rb:27:in `compile'
from /usr/local/bundle/bundler/gems/metanorma-cli-eb2534021b79/lib/metanorma/cli/compiler.rb:45:in `compile_file'
from /usr/local/bundle/bundler/gems/metanorma-cli-eb2534021b79/lib/metanorma/cli/compiler.rb:30:in `compile'
from /usr/local/bundle/bundler/gems/metanorma-cli-eb2534021b79/lib/metanorma/cli/compiler.rb:35:in `compile'
from /usr/local/bundle/bundler/gems/metanorma-cli-eb2534021b79/lib/metanorma/cli/command.rb:42:in `compile'
from /usr/local/bundle/gems/thor-1.0.1/lib/thor/command.rb:27:in `run'
from /usr/local/bundle/gems/thor-1.0.1/lib/thor/invocation.rb:127:in `invoke_command'
from /usr/local/bundle/gems/thor-1.0.1/lib/thor.rb:392:in `dispatch'
from /usr/local/bundle/gems/thor-1.0.1/lib/thor/base.rb:485:in `start'
from /usr/local/bundle/bundler/gems/metanorma-cli-eb2534021b79/lib/metanorma/cli.rb:62:in `start'
from /usr/local/bundle/bundler/gems/metanorma-cli-eb2534021b79/exe/metanorma:20:in `<top (required)>'
from /usr/local/bundle/bin/metanorma:23:in `load'
from /usr/local/bundle/bin/metanorma:23:in `<main>'
/usr/local/bundle/gems/fontist-1.8.4/lib/fontist/index.rb:7:in `from_yaml': Please fetch index with `fontist update`. (Fontist::Errors::FormulaIndexNotFoundError)
from /usr/local/bundle/gems/fontist-1.8.4/lib/fontist/formula.rb:28:in `find_fonts'
from /usr/local/bundle/gems/fontist-1.8.4/lib/fontist/formula_paths.rb:30:in `find_styles_by_formulas'
from /usr/local/bundle/gems/fontist-1.8.4/lib/fontist/formula_paths.rb:10:in `find'
from /usr/local/bundle/gems/fontist-1.8.4/lib/fontist/system_font.rb:75:in `find_by_formulas'
from /usr/local/bundle/gems/fontist-1.8.4/lib/fontist/system_font.rb:67:in `find_styles'
from /usr/local/bundle/gems/fontist-1.8.4/lib/fontist/system_font.rb:55:in `find_with_name'
from /usr/local/bundle/gems/fontist-1.8.4/lib/fontist/system_font.rb:44:in `find_with_name'
from /usr/local/bundle/gems/fontist-1.8.4/lib/fontist/manifest/locations.rb:64:in `find_font_with_name'
from /usr/local/bundle/gems/fontist-1.8.4/lib/fontist/manifest/install.rb:14:in `file_paths'
from /usr/local/bundle/gems/fontist-1.8.4/lib/fontist/manifest/locations.rb:51:in `block in style_paths'
from /usr/local/bundle/gems/fontist-1.8.4/lib/fontist/manifest/locations.rb:50:in `map'
from /usr/local/bundle/gems/fontist-1.8.4/lib/fontist/manifest/locations.rb:50:in `style_paths'
from /usr/local/bundle/gems/fontist-1.8.4/lib/fontist/manifest/locations.rb:45:in `style_paths_map'
from /usr/local/bundle/gems/fontist-1.8.4/lib/fontist/manifest/locations.rb:40:in `block in font_paths'
from /usr/local/bundle/gems/fontist-1.8.4/lib/fontist/manifest/locations.rb:38:in `each'
from /usr/local/bundle/gems/fontist-1.8.4/lib/fontist/manifest/locations.rb:38:in `map'
from /usr/local/bundle/gems/fontist-1.8.4/lib/fontist/manifest/locations.rb:38:in `font_paths'
from /usr/local/bundle/gems/fontist-1.8.4/lib/fontist/manifest/locations.rb:26:in `call'
from /usr/local/bundle/gems/fontist-1.8.4/lib/fontist/manifest/locations.rb:21:in `from_hash'
from /usr/local/bundle/gems/metanorma-1.2.4/lib/metanorma/compile.rb:312:in `fontist_install'
from /usr/local/bundle/gems/metanorma-1.2.4/lib/metanorma/compile.rb:293:in `install_fonts_safe'
from /usr/local/bundle/gems/metanorma-1.2.4/lib/metanorma/compile.rb:287:in `install_fonts'
from /usr/local/bundle/gems/metanorma-1.2.4/lib/metanorma/compile.rb:27:in `compile'
from /usr/local/bundle/bundler/gems/metanorma-cli-eb2534021b79/lib/metanorma/cli/compiler.rb:45:in `compile_file'
from /usr/local/bundle/bundler/gems/metanorma-cli-eb2534021b79/lib/metanorma/cli/compiler.rb:30:in `compile'
from /usr/local/bundle/bundler/gems/metanorma-cli-eb2534021b79/lib/metanorma/cli/compiler.rb:35:in `compile'
from /usr/local/bundle/bundler/gems/metanorma-cli-eb2534021b79/lib/metanorma/cli/command.rb:42:in `compile'
from /usr/local/bundle/gems/thor-1.0.1/lib/thor/command.rb:27:in `run'
from /usr/local/bundle/gems/thor-1.0.1/lib/thor/invocation.rb:127:in `invoke_command'
from /usr/local/bundle/gems/thor-1.0.1/lib/thor.rb:392:in `dispatch'
from /usr/local/bundle/gems/thor-1.0.1/lib/thor/base.rb:485:in `start'
from /usr/local/bundle/bundler/gems/metanorma-cli-eb2534021b79/lib/metanorma/cli.rb:62:in `start'
from /usr/local/bundle/bundler/gems/metanorma-cli-eb2534021b79/exe/metanorma:20:in `<top (required)>'
from /usr/local/bundle/bin/metanorma:23:in `load'
from /usr/local/bundle/bin/metanorma:23:in `<main>'

<wnload these fonts for you? => TYPE 'Yes' or 'No': 
================
```

**Comment by ronaldtse** [comment]
I'm getting this same problem in 1.8.4. This problem did not exist in 1.8.3.

```sh
$ fontist install arial
Fonts found at:
- /Library/Fonts/Microsoft/Arial Bold.ttf
- /Library/Fonts/Microsoft/Arial.ttf
- /Library/Fonts/Microsoft/Arial Bold Italic.ttf
- /Library/Fonts/Microsoft/Arial Italic.ttf
- /System/Library/Fonts/Supplemental/Arial Bold.ttf
- /System/Library/Fonts/Supplemental/Arial.ttf
- /System/Library/Fonts/Supplemental/Arial Bold Italic.ttf
- /System/Library/Fonts/Supplemental/Arial Italic.ttf
- /Applications/Microsoft Word.app/Contents/Resources/DFonts/arialbd.ttf
- /Applications/Microsoft Word.app/Contents/Resources/DFonts/arial.ttf
- /Applications/Microsoft Word.app/Contents/Resources/DFonts/ariblk.ttf
- /Applications/Microsoft Word.app/Contents/Resources/DFonts/ariali.ttf
- /Applications/Microsoft Word.app/Contents/Resources/DFonts/arialbi.ttf
- /Applications/Microsoft Excel.app/Contents/Resources/DFonts/arialbd.ttf
- /Applications/Microsoft Excel.app/Contents/Resources/DFonts/arial.ttf
- /Applications/Microsoft Excel.app/Contents/Resources/DFonts/ariblk.ttf
- /Applications/Microsoft Excel.app/Contents/Resources/DFonts/ariali.ttf
- /Applications/Microsoft Excel.app/Contents/Resources/DFonts/arialbi.ttf
- /Applications/Microsoft Outlook.app/Contents/Resources/DFonts/arialbd.ttf
- /Applications/Microsoft Outlook.app/Contents/Resources/DFonts/arial.ttf
- /Applications/Microsoft Outlook.app/Contents/Resources/DFonts/ariblk.ttf
- /Applications/Microsoft Outlook.app/Contents/Resources/DFonts/ariali.ttf
- /Applications/Microsoft Outlook.app/Contents/Resources/DFonts/arialbi.ttf
- /Applications/Microsoft OneNote.app/Contents/Resources/DFonts/arialbd.ttf
- /Applications/Microsoft OneNote.app/Contents/Resources/DFonts/arial.ttf
- /Applications/Microsoft OneNote.app/Contents/Resources/DFonts/ariblk.ttf
- /Applications/Microsoft OneNote.app/Contents/Resources/DFonts/ariali.ttf
- /Applications/Microsoft OneNote.app/Contents/Resources/DFonts/arialbi.ttf
- /Applications/Microsoft PowerPoint.app/Contents/Resources/DFonts/arialbd.ttf
- /Applications/Microsoft PowerPoint.app/Contents/Resources/DFonts/arial.ttf
- /Applications/Microsoft PowerPoint.app/Contents/Resources/DFonts/ariblk.ttf
- /Applications/Microsoft PowerPoint.app/Contents/Resources/DFonts/ariali.ttf
- /Applications/Microsoft PowerPoint.app/Contents/Resources/DFonts/arialbi.ttf
```

```sh
$ fontist status arial
'arial' font is missing, please run `fontist install 'arial'` to download the font.
```

**Comment by alexeymorozov** [comment]
1. Found the cause of inability to detect Arial even when it's installed from a formula. Will fix.
2. There is also issue. The `status` command wasn't designed to detect system-installed fonts. Would we like it to? Can do, just need to better understand the needs.

**Comment by ronaldtse** [comment]
> Found the cause of inability to detect Arial even when it's installed from a formula. Will fix.

Thanks.

> There is also another issue. The status command wasn't designed to detect system-installed fonts. Would we like it to? Can do, just need to better understand the needs. Now the manifest-locations command does this thing.

Yes please. The status command should also detect system-installed fonts because there is no point in installing a font that already exists and users cannot tell the difference.

**Comment by alexeymorozov** [comment]
Ok, let's do it. `status` can mimic the install logic, so its output can be:

```
$ fontist status arial
Fonts found at:
- /System/Library/Fonts/Supplemental/Arial Bold.ttf
- /System/Library/Fonts/Supplemental/Arial.ttf
- /System/Library/Fonts/Supplemental/Arial Bold Italic.ttf
- /System/Library/Fonts/Supplemental/Arial Italic.ttf
- /Users/amorozov/.fontist/fonts/arialbd.ttf (from webcore formula)
- /Users/amorozov/.fontist/fonts/arial.ttf (from webcore formula)
- /Users/amorozov/.fontist/fonts/ariali.ttf (from webcore formula)
- /Users/amorozov/.fontist/fonts/arialbi.ttf (from webcore formula)
```

**Comment by ronaldtse** [comment]
Yes the logic for both commands should be the same. Thanks! When it’s done please close this task.

**Comment by alexeymorozov** [comment]
Released in v1.8.5 🚀 

Closing
